examples/form/mantine/useModalForm/src/pages/posts/list.tsx

```diff
@@ -0,0 +1,85 @@
+import { useTable } from "@pankod/refine-core";
+import {
+    EditButton,
+    List,
+    Pagination,
+    Table,
+    useModalForm,
+} from "@pankod/refine-mantine";
+
+import { CreatePostModal, EditPostModal } from "../../components";
+import { IPost } from "../../interfaces";
+
+export const PostList: React.FC = () => {
+    const { tableQueryResult, current, setCurrent, pageCount } =
+        useTable<IPost>({
+            resource: "posts",
+        });
```

Comment:
```suggestion
        useTable<IPost>();
```
is it necessary to add a resource here?